### PR TITLE
Bugs/46/121 i18n wire forms fix

### PR DESCRIPTION
**Pull request by gabrieldiac11**
Tasks:
1. [https://onramper.atlassian.net/browse/ONRMP-120](https://onramper.atlassian.net/browse/ONRMP-120)

2. [https://onramper.atlassian.net/browse/ONRMP-121](https://onramper.atlassian.net/browse/ONRMP-121)
- this is also related to a PR from the server side (server and widget SHOULD BE RUN TOGETHER)
[https://github.com/onramper/server/pull/54](https://github.com/onramper/server/pull/54)

### CODE REVIEW

File: package/src/steps/FormView/BodyFormView.tsx [hunk 1]
```diff
@@ -281,9 +281,9 @@ const BodyFormView: React.FC<BodyFormViewType> = (props) => {
                             ))
                             || ((field.type === 'boolean' && field.name === 'termsOfUse') && (
                                 <label key={i} className={`${stylesCommon.body__child} ${styles.terms}`}>
-                                    <input type="checkbox" checked={collected[field.name] ?? false} name={field.name} onChange={(e) => onChange(e.currentTarget.name, e.currentTarget.checked, e.currentTarget.type)} />&nbsp;I accept {
+                                    <input type="checkbox" checked={collected[field.name] ?? false} name={field.name} onChange={(e) => onChange(e.currentTarget.name, e.currentTarget.checked, e.currentTarget.type)} />&nbsp;{t('formView.termsAgreement')} {
                                         field.terms?.map<React.ReactNode>((term, i) => <a key={i} href={term.url} target='_blank' rel="noopener noreferrer">{term.humanName}</a>)
-                                            .reduce((acc, actual, i, arr) => [acc, i === arr.length - 1 ? ' and ' : ', ', actual])
+                                            .reduce((acc, actual, i, arr) => [acc, i === arr.length - 1 ? ` ${t('misc.and')} ` : ', ', actual])
```
Comment (Noorts) [inline package/src/steps/FormView/BodyFormView.tsx:286]: This could be simplified to `t('misc.and')` right?

**Review by Noorts — CHANGES_REQUESTED** [review]

**Review by Noorts — CHANGES_REQUESTED** [review]
Good work!

The payment review screen (Wyre) has almost been translated completely. One of the only things still left untranslated is the header "Payment review".

Has the screen in the Wyre flow after the screen containing the Terms of Use also been translated? Please verify.

File: package/src/steps/FormView/BodyFormView.tsx (hunk 1 above)
Comment (gabrieldiac11) [reply package/src/steps/FormView/BodyFormView.tsx:286]: You mean to get rid of spaces? I've added the spaces before aned after the 'and', rather then including them in the translation value

**Comment by gabrieldiac11** [comment]
> Has the screen in the Wyre flow after the screen containing the Terms of Use also been translated? Please verify.

that part is really a hybrid between widget and server, to have the full picture please use the server using the branch bugs/46/121-i18n-wire-forms-fix
I think it may be related to this

File: package/src/steps/FormView/BodyFormView.tsx (hunk 1 above)
Comment (Noorts) [reply package/src/steps/FormView/BodyFormView.tsx:286]: Smart 👍

**Review by Noorts — APPROVED** [review]

**Pull request merged**